jobs/update-colin-filings/config.py

```diff
@@ -47,12 +47,13 @@ class _Config(object):  # pylint: disable=too-few-public-methods
     PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 
     COLIN_URL = os.getenv('COLIN_URL', '')
-    LEGAL_URL = os.getenv('LEGAL_URL', '')
+    LEGAL_API_URL = os.getenv('LEGAL_API_URL', '')
```

Comment:
moving to v2 endpoints. creating a new env variable as `LEGAL_URL` is used in future-effective job (which need more update, in separate ticket).